ArkEcosystem/laravel-ui
# fix: password visibility toggle

## Summary
https://app.clickup.com/t/pqenbt

This PR fixes "password toggle" component by removing a non-standard CSS rule. 
Instead it defines a default `autocomplete="current-password"` attribute. This informs modern browsers that we asking for a current password and it can be handy with 3rd-party password managers. In this case browsers may not ask you to store password (but sometimes it depends on browser's configuration).

Another possible solution is to use `autocomplete="one-time-code" but this can cause problem with 3rd-party password managers

## How to test
1. merge this PR into Nodem
2. login and go to `/user/settings/security`
3. enable 2FA
4. then click to disable 2FA
5. a modal appears asking you to fill in your current password
6. play with that modal to see if:
   * toggle is working properly
   * browser don't ask to store the typed password
   * browser don't propose you a new password

## Checklist
